docs/state-channels/fsm_initiator.puml

```diff
@@ -0,0 +1,57 @@
+@startuml
```

Comment:
Would it be possible to automate as Makefile the generation of this plantuml file, and hook that to CI?